src/nft/components/bag/BagFooter.tsx

```diff
@@ -199,6 +203,17 @@ export const BagFooter = ({
   const isPending = PENDING_BAG_STATUSES.includes(bagStatus)
   const activeCurrency = inputCurrency ?? defaultCurrency
 
+  const parsedAmount = useMemo(() => {
+    if (!inputCurrency) return undefined
+    return tryParseCurrencyAmount(formatEther(totalEthPrice.toString()), defaultCurrency ?? undefined)
```

Comment:
should the second input for `tryParseCurrencyAmount` be inputCurrency? I'm a little confused why it's not used by this function

Reply:
this is just to convert the total amount of eth to a CurrencyAmount not the inputCurrency

Reply:
basically this function takes in a string and the currency you want. so like '78.2' and 'ETH' and it tries to create a `CurrencyAmount` from that input